### DISCUSSION TITLE
Add a message parameter to the `{% exit %}` tag

### DISCUSSION THREAD

**Discussion by mmikkel** [craftcms/cms#13166] [Ideas] [answered]
When using the `{% exit %}` tag, I've often wished that it was possible to set an HTTP exception message (like it is when throwing something like `BadRequestHttpException` from PHP), i.e. something like this:

```twig
{% exit 400 'Some message' %}
```

**Comment by brandonkelly** [comment, marked as answer]
Added for Craft 4.5! (cdc03996973634a820429eb65997da8f49dc2291)

**Reply by mmikkel** [reply]
Amazing! Thanks a lot Brandon!

**Reply by brandonkelly** [reply]
Craft 4.5.0 is out with that!
